projects/aca-content/src/lib/components/knowledge-retrieval/search-ai/search-ai-results/search-ai-results.component.ts

```diff
@@ -192,4 +199,22 @@ export class SearchAiResultsComponent extends PageComponent implements OnInit, O
         () => (this._hasAnsweringError = true)
       );
   }
+
+  private aiSearchRetryWhen<T>(errors: Observable<T>): Observable<T> {
+    const delayBetweenRetries = 3000;
+    const maxRetries = 9;
+
+    return errors.pipe(
+      skipWhile(() => this._hasAnsweringError),
```

Comment:
There is getter for hasAnsweringError field so please use getter for referencing to that